pallets/handles/src/handles-utils/src/tests/suffix_tests.rs

```diff
@@ -0,0 +1,19 @@
+use crate::suffix::{generate_seed, generate_unique_suffixes};
+
+#[test]
+fn should_always_have_the_same_seed() {
+	assert_eq!(generate_seed("abcdefg"), 15079896798642598352u64);
+	assert_eq!(generate_seed("gfedcba"), 9497970330616778036u64);
+}
+
+#[test]
+fn should_generate_the_same_sequence() {
```

Comment:
Awesome! I am really surprised that there were no failing tests. I would have expected some extrinsic test failure for the stored handle - CanonicalBaseHandleToSuffixIndex. I recommend a different pr adding some more tests around this.